wandb/sdk/wandb_run.py

```diff
@@ -1066,6 +1071,9 @@ def _set_run_obj(self, run_obj: RunRecord) -> None:
         self._run_obj = run_obj
         self._entity = run_obj.entity
         self._project = run_obj.project
+        self._settings.update(
```

Comment:
probably won't need it after my PR since I'm adding an `_apply_run_start` stage to init...

Reply:
Is this needed anymore?   @kptkin PR was merged.